engine/runtime-parser/src/main/java/org/enso/compiler/core/ir/JName.java

```diff
@@ -18,7 +19,11 @@ JName duplicate(
       boolean keepDiagnostics,
       boolean keepIdentifiers);
 
-  interface JBlank extends JName {}
+  interface JBlank extends JName {
```

Comment:
These sample `JXyz` interfaces should probably be moved outside of real code to tests.

Reply:
Done in c8cbc98dc339eb509a0db102794b20570a3070ba